requirements.txt

```diff
@@ -1,2 +1,5 @@
-django 3.1.7
-python 3.9
+Django==3.1.7
+django-heroku==0.3.1
+gunicorn==20.0.4
+selenium==3.141.0
+webdriver-manager==3.3.0
```

Comment:
I think we should have requirements as >= rather than strictly ==; if we update Django, for example some minor patch, our website won't be compatible to run.

Reply:
yep will update that